doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -299,7 +299,8 @@ NFC samples
 nRF5340 samples
 ---------------
 
-|no_changes_yet_note|
+* The ``multiprotocol_rpmsg`` sample is removed.
+  The :ref:`ipc_radio` application should be used instead.
```

Comment:
```suggestion
* Removed the ``nRF5340: Multiprotocol RPMsg`` sample.
  Use the :ref:`ipc_radio` application instead.
```

Reply:
Also take out the link to the removed sample from the various release notes and migration guide, as it breaks the doc build.

Reply:
@peknis , I addressed (hopefully) above two issues you pointed, please check